### ISSUE TITLE
When selecting US, Malaysia information is displayed.

### ISSUE THREAD

**Issue by aillynesc** [CSSEGISandData/COVID-19#1195]
There's a bug when selecting US country, displays the number of Malaysia in the Total Confirmed, Total Deaths and Total Recovered boxes.
Please see the snapshot. 
![image](https://user-images.githubusercontent.com/43277199/77236967-8b511c00-6b91-11ea-9f1c-74f311f07fb4.png)

**Comment by MilanDonhowe** [comment]
Do you still have the issue?  When I open up the web-page the US related numbers show up when I click "US": ![image of US numbers](https://i.imgur.com/M7x89iu.png)

**Comment by aillynesc** [comment]
No. I refreshed my browser and I can see now the correct information for
US. Thank you.
